data/categories/sg_sporting_goods.yml

```diff
@@ -6935,6 +6937,167 @@
   attributes:
   - color
   - pattern
+- id: sg-4-1-10
+  name: Wingfoils
+  children:
+  - sg-4-1-10-1
+  - sg-4-1-10-2
+  - sg-4-1-10-3
+  - sg-4-1-10-4
+  - sg-4-1-10-5
+  attributes: []
```

Comment:
I suggest we keep consistent naming for the area. This is called "Wingfoils" (in plural) and the next section "Windfoil" (in singular). Should we call them "Wingfoiling" and "Windfoiling" for consistency?